main.tf

```diff
@@ -21,4 +21,6 @@ module "aws_lambda_function" {
   memory_size               = var.memory_size
   efs_arn                   = var.efs_arn
   efs_mount_path            = var.efs_mount_path
+  access_key_id_glm_arn     = var.access_key_id_glm_arn
```

Comment:
Why do we still need the variables here? we should read them (if applicable from the event)